src/pages/ServiceDetails/ServiceInfo/ServiceInfoVirtualServices.css.map

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we commit the .map files ?
Those are for debugging ?
I don't know just wonder what is the usual rule.

Reply:
yep, I totally forgot to remove them. Probably, we should update .gitignore to ignore all .map files